src/components/Input/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+
+// Custom hooks
+import { useAppSelector } from '../../redux/hooks';
+
+// Styles
+import * as S from './styles';
+
+const INPUT = {
+  input: ({ type, placeholder, theme }: { type: string; placeholder: string; theme: string }) => (
+    <S.Input type={type} placeholder={placeholder} theme={theme} />
+  )
```

Comment:
just interested - why do you need to create the separate const for input?

Reply:
My initial idea was to use switch statement, but as I learned from previous training that it is not advised, I decided to use declarative manner.